core/templates/dev/head/domain/topic/TopicUpdateService.js

```diff
@@ -199,30 +199,30 @@ oppia.factory('TopicUpdateService', [
           topic.addSubtopic(title);
         }, function(changeDict, topic) {
           // Undo.
-          var subtopicId = _getParameterFromChangeDict(
-            changeDict, 'subtopic_id');
+          var subtopicId = _getSubtopicIdFromChangeDict(changeDict);
           topic.deleteSubtopic(subtopicId);
         });
       },
 
       /**
        * Deletes a subtopic from the topic and records the change in
-       * the undo/redo service.
+       * the undo/redo service.The argument isNewlyCreated should be passed from
+       * the calling function, after checking the changelist to see if this
+       * subtopic was created in the same changelist.
```

Comment:
This is a bit vague, consider adding a standard-style jsdoc explaining the types of each arg and what each arg represents.

Reply:
Done.